web/src/todo_widget.js

```diff
@@ -179,8 +179,8 @@ export function activate(opts) {
                 return;
             }
 
-            $elem.find(".add-task").val("").trigger("focus");
-            $elem.find(".add-desc").val("").trigger("focus");
+            $elem.find("input.add-desc").val("").trigger("focus");
+            $elem.find("input.add-task").val("").trigger("focus");
```

Comment:
Swapping the order seems like an improvement but it's frankly crazy that we focus both of these elements in order; we should remove the `.focus` on the wrong one.